elastio-stream-oracle-database-backup/backup-oracle-table.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Please write a `README.md` that explains this script and how to use it.

In particular you should mention that you assume there is a `.env` file that is a valid shell script with environment variables defined.